### ISSUE TITLE
Give Proper Workflow of the API

### ISSUE THREAD

**Issue by shubhamjha25** [suyalamritanshu/Social-Media-Api#9]
A proper structure of how this API works should be given in the documentation.
Workflow should include from which routes to begin with, how to proceed after authentication etc. or in simple words How to completely fetch this API.

It will help those who are willing to contribute to this project and will also save the time for those who are willing to consume this ready-made API on their frontend.

@suyalamritanshu Do have a look on this issue.

P.S.: Don't assign me on this.

**Comment by suyalamritanshu** [comment]
Idea is nice, do you want to do this?

**Comment by shubhamjha25** [comment]
No, I am also facing some problems while understanding the codebase. 😝
That's why raised this issue.

Leaving it open for someone else to do it.

**Comment by suyalamritanshu** [comment]
Cool, this issue is opened for all
